Source/Fuse.Marshal/Marshal.Cast.uno

```diff
@@ -114,6 +114,75 @@ namespace Fuse
 
 			throw new MarshalException(o, typeof(float4));
 		}
+		
+		/**
+			Converts a value to a float4
+			
+			@return true if converted successfully, false if no suitable conversion exists
+			@param value the result value (0 padded as necessary)
+			@param size the size of the result
+		*/
+		public static bool TryMarshalToFloat4(object o, out float4 value, out int size)
```

Comment:
The `ToFloat4` takes a single value and does `float4(v)` whereas the new `TryToZeroFloat4` (new name) does `float4(v,0,0,0)`. Similarly for float2/3